README.md

```diff
@@ -1,2 +1,115 @@
+    <img alt="MIT Licensed" src="https://img.shields.io/github/license/Hamzaoutdoors/to-do-list?style=flat-square">
+  </a>
+  <a href="https://github.com/Hamzaoutdoors/Catalog-Of-My-Things">
+    <img alt="GitHub last commit (branch)" src="https://img.shields.io/github/last-commit/Hamzaoutdoors/Catalog-Of-My-Things/dev?color=blue&style=flat-square">
+  </a>
+  <a href="https://github.com/Hamzaoutdoors/Catalog-Of-My-Things">
+    <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/Hamzaoutdoors/Catalog-Of-My-Things?color=green&label=%E2%98%85%20stars%20&style=flat-square">
+  </a>
+  <a href="https://github.com/Hamzaoutdoors">
+    <img alt="GitHub followers" src="https://img.shields.io/github/followers/Hamzaoutdoors?color=yellow&logo=github&style=flat-square">
+  </a>
+</p>
+
+<div align="center">
+ <img alt="Microverse" src="https://img.shields.io/badge/Ubuntu-E95420?style=for-the-badge&logo=ubuntu&logoColor=white"> <img src="https://img.shields.io/badge/postgres-%23316192.svg?style=for-the-badge&logo=postgresql&logoColor=white"/>  <img src="https://img.shields.io/badge/mysql-%2300f.svg?style=for-the-badge&logo=mysql&logoColor=white"/>  <img src="https://img.shields.io/badge/ruby-%23CC342D.svg?style=for-the-badge&logo=ruby&logoColor=white"/> <img src="https://img.shields.io/badge/rails-%23CC0000.svg?style=for-the-badge&logo=ruby-on-rails&logoColor=white"/></div>
+
+   </br>
+ <div align="center">
+  <em align="center" style>" Everyday life is like programming, I guess. If you love something you can put beauty into it."</em><strong> - Sabrina Bryan</strong>
+  </div>
+   </br>
+
+# Ruby Group Capstone : Catalog of my things
+
+In this project, we create a console app that will help you to keep a record of different types of things you won: 
+ - **Books**
+ - **Music albums**
+ - **Games** 
+
+Everything will be based on the UML class diagram presented below. The data will be stored in JSON files, furthermore, we prepared a database with tables structure analogical to the program's class structure.
+
+</br>
+ <div align="center">
+   <kbd>
+    <img alt="gif" src="https://user-images.githubusercontent.com/80895497/143606720-cc2cd9bd-2909-44f0-b2ea-8056741679c4.gif" width=100% />
+    <p>Catalog of my things Console App</p>
+   </kbd>
+</div>
+
+</br>
+ <div align="center">
+    <img alt="gif" src="./catalog_of_my_things.png" width=100% />
+    <p>UML class diagram of Catalog of my things project</p>
+</div>
+
+## Getting Started
+To get a local copy up and running follow these simple example steps :
+
+ - First of all make sure you have `Ruby` installed in your machine
+ - clone the project with the following command line : 
+```
+$ git clone git@github.com:aliabbani/Catalog-of-my-things.git
+```
+
+ - Then go to the right folder 
+```
+$ cd Catalog-of-my-things
+```
+ - Start using the app by runing `ruby main.rb` on your terminal
+ - Follow the app instructions.
```

Comment:
- [ ]  [OPTIONAL] Your README looks so professional however please add the instructions on how  to run the test

Reply:
Thanks for the feedback! We will do it.

![](https://media3.giphy.com/media/oe8Ii2ZyKl1fy/giphy.gif)